main.go

```diff
@@ -49,13 +51,20 @@ func generatorsHelp() string {
 	return buffer.String()
 }
 
+func isPipe() bool {
+	stat, _ := os.Stdin.Stat()
```

Comment:
I didn't check during testing if the method works and then forgot to handle it. I'll add error checking and exit if there is an error with reading pipes (makes sense).